homeassistant/components/mobile_app/binary_sensor.py

```diff
@@ -39,7 +39,7 @@ async def async_setup_entry(
     for entry in entries:
         if entry.domain != ENTITY_TYPE or entry.disabled_by:
             continue
-        config = {
+        config: dict = {
```

Comment:
Are all the keys not strings?
```suggestion
        config: dict[str, Any] = {
```